src/test/java/com/fasterxml/jackson/databind/jsontype/JsonTypeInfoSimpleClassName4061TestContainer.java

```diff
@@ -0,0 +1,16 @@
+package com.fasterxml.jackson.databind.jsontype;
+
+import com.fasterxml.jackson.databind.BaseMapTest;
+
+/**
+ * Test utility for {@link JsonTypeInfoSimpleClassName4061Test#testDuplicateNameLastOneWins()}.
+ * 
+ * @since 2.16
+ */
+public class JsonTypeInfoSimpleClassName4061TestContainer extends BaseMapTest
```

Comment:
Should not extend `BaseMapTest` when there are no tests contained.

But wouldn't it be possible to have 2 classes in main test class itself: one non-public at main-level, another one nested within test class?